doc/whats_new.rst

```diff
@@ -75,6 +75,8 @@ Changelog
 Bug
 ~~~
 
+- Fix reading edf file annotations by `Joan Massich`_ (`#5580 <https://github.com/mne-tools/mne-python/pull/5580>`_)
```

Comment:
-1 on adding these for 0.17, let's start doing this for 0.18 so they can be for every one. Unless you feel like going back through all of the existing ones the inconsistency probably isn't worth the added information